packages/sdk/src/actions/simulateDepositTx.ts

```diff
@@ -91,9 +91,9 @@ export async function simulateDepositTx(params: SimulateDepositTxParams) {
 
   const result = await publicClient.simulateContract({
     account: walletClient.account,
-    abi: spokePoolAbi,
+    abi: spokePoolAbiV4,
     address: spokePoolAddress,
-    functionName: useExclusiveRelayer ? "depositExclusive" : "depositV3",
```

Comment:
I would keep this logic for now. Because the API will be updated later than the app-sdk.

Reply:
Resolved offline